progs/get-subseqs/main.go

```diff
@@ -18,7 +21,30 @@ func main() {
 	}
 	defer fastaFile.Close()
 
-	fmt.Println("I should be processing the fasta file now!")
+	//########################################################
+	sequences := make([]string, 0)
+	fileScanner := bufio.NewScanner(fastaFile)
+
+	//scan file line by line
+	for fileScanner.Scan() {
+		line := fileScanner.Text()
+
+		if len(line) == 0 {
+			continue
+		}
+
+		//handle 'name of sequence' cases
+		if line[0:1] == ">" {
+			sequences = append(sequences, "")
+			sequences[len(sequences)-1] = strings.TrimSpace(line[1:])
```

Comment:
Why append an empty string and then overwrite it? Couldn't you just append the header?

```go
sequences = append(sequences, strings.TrimSpace(line[1:]))
```